platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/uptime/UptimeMetrics.java

```diff
@@ -61,8 +61,7 @@ class UptimeMetrics {
             .withDescription("The time, in microseconds, required to compute uptime information each round.");
     private final RunningAverageMetric uptimeComputationTime;
 
-    private static final String ROUNDS_SINCE_LAST_CONSENSUS_EVENT = "roundsSinceLastConsensusEvent-";
-    private static final String ROUNDS_SINCE_LAST_JUDGE = "roundsSinceLastJudge-";
+    private static final String ROUNDS_SINCE_LAST_CONSENSUS_EVENT = "roundsSinceLastConsensusEvent_";
```

Comment:
We really need to check after the merge that all the grafana dashboards still work

Reply:
True that, although I haven't found any difference between the resulting metric names and the values in graphana json other than the title of the metric.

Reply:
From my perspective, it's not the end of the world if a metric name changes. Most of our current metric names are not so great, and I have intentions to change many of them. 😜